docs/quick-start.md

```diff
@@ -0,0 +1,92 @@
+- nvm: `curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.0/install.sh | bash`
+
+The Jetpack Monorepo requires the following to be installed on your machine:
+
+- Node.js (used for build process and our CLI): `nvm install`
+	- You can see the current latest supported version of Node.js in the monorepo root's [`package.json`](https://github.com/Automattic/jetpack/blob/master/package.json) under `engines`
+	- You can switch versions of node using nvm: `nvm ls` to list installed versions of node, `nvm use [version number]` to switch.
+- PNPM (a Node.js package manager): `npm install -g pnpm`
+- Composer (our PHP package manager): `brew install composer`
+- PHP (the language at the core of the WordPress ecosystem): `brew install php@8.0`
+- Once the pre-requisites are installed, you can clone the GitHub repo:
+	- `git clone https://github.com/Automattic/jetpack.git`
+	- If you're not an Automattician, you can [fork the repo following the instructions here](https://docs.github.com/en/get-started/quickstart/contributing-to-projects).
+- Jetpack CLI (an internal tool that assists with development): `pnpm install && pnpm cli-setup`
+	- [You can read more about using the CLI here](https://github.com/Automattic/jetpack/blob/master/tools/cli/README.md).
+
+## Running Jetpack locally
+
+After everything is installed, you're ready to run Jetpack locally! While there are other supported methods of doing this, we recommend and support using Docker containers. 
+
+To setup Docker:
+- Run `brew install --cask docker` or [download the latest version here](https://www.docker.com/products/docker-desktop). 
+- Copy the settings file so you can modify passwords and other configurations: `cp tools/docker/default.env tools/docker/.env`
+- Start the Docker container using `jetpack docker up`
+- Install WordPress in your Docker container using `jetpack docker install` 
+- Open up http://localhost to see your site!
+
+For more in depth docker instructions, follow the [Docker environment for Jetpack Development guide](../tools/docker/README.md).
+
+### Setting up Jurassic Tube
+
+To test network related features of Jetpack, you'll need a test site that can create local HTTP tunnels. If you're an Automattician, we recommend using Jurassic Tube:
+
+- Add a subdomain on [jurassic.tube](https://jurassic.tube/)
+- Make sure Docker is running `jetpack docker up -d`
+- Make sure you've run `pnpm install && pnpm cli-link`
+- Stand on the monorepo root in your terminal and run `mkdir tools/docker/bin/jt`
+- Run the installation script: `chmod +x tools/docker/bin/jt/installer.sh && tools/docker/bin/jt/installer.sh`
+- Use `jetpack docker jt-up your-username custom-subdomain` to start the site.
+- Your site should be avalable at `https://custom-subdomain.jurassic.tube`
+
+*Optional*
+
+To make things easier, you can configure Jurassic Tube to remember your username and subdomain.
```

Comment:
For this guide, I don't think this should be optional. We should be opinionated to make their dev experience the best possible.

Reply:
Agreed, consolidated those sections.